docfx.json

```diff
@@ -314,7 +314,6 @@
                 "docs/core/extensions/**/**.md": "IEvangelist",
                 "docs/core/docker/**/**.md": "IEvangelist",
                 "docs/core/native-interop/**/**.md": "jkoritzinsky",
-                "docs/core/porting/**/**.md": "cartermp",
```

Comment:
@gewarren 

I tagged you on this one, because the up-folder default is you. Is that right, or should it be someone else?

Reply:
@BillWagner I'm fine either way.